sblim-sfcc.rb

```diff
@@ -8,11 +6,18 @@ class SblimSfcc < Formula
   depends_on :libtool
   depends_on :autoconf
   depends_on :automake
-  
+  depends_on :openssl
+
   def install
     system "./configure","--prefix=#{prefix}", "--disable-dependency-tracking"
     system "make", "install"
   end
+
+  def test
+    # no executables, test to see if the man page has been installed
+    system "man -W CMCIClient"
```

Comment:
thanks for pointing us in the right direction @mikemcquaid - I updated these two tests per you suggestions - https://github.com/jameskyle/homebrew-rejected/pull/6